package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@axelar-network/interchain-token-service",
-  "version": "2.0.1",
+  "version": "2.1.0",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "@axelar-network/interchain-token-service",
-      "version": "2.0.1",
+      "version": "2.1.0",
```

Comment:
why this change? did you not branch off main? Looks like your PR has merge conflicts, can you resolve them?